tendrl/commons/utils/ansible_module_runner.py

```diff
@@ -1,9 +1,12 @@
 import os
 import subprocess
+import sys
 import tempfile
 
 import ansible.executor.module_common as module_common
 from ansible import modules
+from ansible.parsing.dataloader import DataLoader
+from ansible.template import Templar
```

Comment:
Should the import variable be called `Templar` or `Template` ?

Reply:
it is templar only @shtripat https://github.com/ansible/ansible/blob/4d3a6123d5b84ed8cdaba4e945e6f1d55c35c4b4/lib/ansible/template/__init__.py#L267

Reply:
Ack.